m4_sessions/1706-inning/group_five.md

```diff
@@ -1,9 +1,50 @@
 # Group Five
 
-### Session Title:
+### Session Title: How and When to Learn a New Language?
 
 ### Session Outcomes: 
+- Create a repo of resources and tutorials for learning new languages and technologies that students can contribute to.
+- Provide outline of our process and experiences
+- Provide space for people to share their own challenges and solutions.
 
 ### Session Outline:
 
-### Names of group members:
+#### Whole Group discussion
+
+##### 10 minute introduction, 
+
+##### 15 min intro
+
+
+- Deliberate Practice
+  - Process of learning
+    - ID a specific subset
+    - Focus on performance in that subset
+    - Get feedback
+    - Update 
+
+   - Motivations
+    - Project
+    - Teaching deadline
```

Comment:
Would love for you to really build this out more...who can you teach  your new language to and which audiences benefit the most from learning from you?  How can you potentially reach out to communities that are important to you to spread that knowledge?